### PR TITLE
added tests for KSV003, KSV011 and KSV015 general rules (part1)

### PR DESCRIPTION
**Pull request by AndreyLevchenko**

### CODE REVIEW

File: kubernetes/policies/general/CPU_not_limited_test.rego [hunk 1]
```diff
@@ -0,0 +1,41 @@
+package appshield.kubernetes.KSV011
+
+test_denied {
```
Comment (yoavrotems) [inline kubernetes/policies/general/CPU_not_limited_test.rego:3]: test_CPU_not_limited_denied

File: kubernetes/policies/general/CPU_not_limited_test.rego [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+			"image": "busybox",
+			"name": "hello",
+		}]},
+	}
+
+	count(r) == 1
+	r[_].msg == "Container 'hello' of Pod 'hello-cpu-limit' should set 'resources.limits.cpu'"
+}
+
+test_allowed {
```
Comment (yoavrotems) [inline kubernetes/policies/general/CPU_not_limited_test.rego:23]: test_CPU_limited_allowed

File: kubernetes/policies/general/CPU_requests_not_specified_test.rego [hunk 3]
```diff
@@ -0,0 +1,41 @@
+package appshield.kubernetes.KSV015
+
+test_denied {
```
Comment (yoavrotems) [inline kubernetes/policies/general/CPU_requests_not_specified_test.rego:3]: test_CPU_requests_not_specified_denied

File: kubernetes/policies/general/CPU_requests_not_specified_test.rego [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+			"image": "busybox",
+			"name": "hello",
+		}]},
+	}
+
+	count(r) == 1
+	r[_].msg == "Container 'hello' of Pod 'hello-cpu-request' should set 'resources.requests.cpu'"
+}
+
+test_allowed {
```
Comment (yoavrotems) [inline kubernetes/policies/general/CPU_requests_not_specified_test.rego:23]: test_CPU_requests_specified_allowed

File: kubernetes/policies/general/capabilities_no_drop_all_test.rego [hunk 5]
```diff
@@ -0,0 +1,41 @@
+package appshield.kubernetes.KSV003
+
+test_denied {
```
Comment (yoavrotems) [inline kubernetes/policies/general/capabilities_no_drop_all_test.rego:3]: test_cap_no_drop_all_denied

File: kubernetes/policies/general/capabilities_no_drop_all_test.rego [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+			"image": "busybox",
+			"name": "hello",
+		}]},
+	}
+
+	count(r) == 1
+	r[_].msg == "Container 'hello' of Pod 'hello-drop-capabilities' should add 'ALL' to 'securityContext.capabilities.drop'"
+}
+
+test_allowed {
```
Comment (yoavrotems) [inline kubernetes/policies/general/capabilities_no_drop_all_test.rego:23]: test_cap_drop_all_allowed

**Review by yoavrotems — CHANGES_REQUESTED** [review]
Looks good just change functions names

**Comment by AndreyLevchenko** [comment]
Hi @yoavrotems 
Thank you for suggestions. I've added changes requested

**Review by yoavrotems — APPROVED** [review]
LGTM

**Pull request merged**